### PR TITLE
New edition

### PR DESCRIPTION
**Pull request by peasant98**
## What this PR does / why we need it

2020, new version yeah.

## Related Issues

Fixes # The fact that there is a new year.

## Some questions

- [X] I have read the contributing guidelines
- [X] I abide by this repository Code of Conduct

## Special notes for your reviewer

(optional)

## Additional Notes

Do you want to add anything else? We :heart: to hear your opinions!

### CODE REVIEW

File: .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910 [hunk 1]
```diff
@@ -0,0 +1 @@
...ts"=>0, "future"=>false, "unpublished"=>false, "whitelist"=>[], "plugins"=>[], "markdown"=>"kramdown", "highlighter"=>"rouge", "lsi"=>false, "excerpt_separator"=>"\n\n", "incremental"=>false, "detach"=>false, "port"=>"4000", "host"=>"127.0.0.1", "baseurl"=>nil, "show_dir_listing"=>false, "permalink"=>"date", "paginate_path"=>"/page:num", "timezone"=>nil, "quiet"=>false, "verbose"=>false, "defaults"=>[], "liquid"=>{"error_mode"=>"warn", "strict_filters"=>false, "strict_variables"=>false}, "kramdown"=>{"auto_ids"=>true, "toc_levels"=>"1..6", "entity_output"=>"as_char", "smart_quotes"=>"lsquo,rsquo,ldquo,rdquo", "input"=>"GFM", "hard_wrap"=>false, "guess_lang"=>true, "footnote_nr"=>1, "show_warnings"=>false}, "original_repo"=>"https://github.com/hackcu/hackcu", "volunteer_mentor_form"=>"https://forms.gle/CczF94ZWjidGAVou9", "live_page"=>"https://live.hackcu.org/", "register"=>"https://forms.gle/QQ2Uh12UD4AkZkrZ8", "description"=>"Join us for HackCU on February 22, 2020 at the University of Colorado Boulder.", "title"=>"HackCU 2020", "sass"=>{"sass_dir"=>"assets/_sass", "style"=>:compressed}, "livereload_port"=>35729, "serving"=>true, "watch"=>true, "url"=>"http://localhost:4000"}:ET
```
Comment (casassg) [inline .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910]: ignore this files m8

File: README.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+HackCU hosted by HackCU website.
 
 # Run project
 
 Needs: Ruby
 
 ## Local 
 
-1. `git clone https://github.com/hackcu/hackcu && cd hackcu`
+1. `git clone  https://github.com/HackCU/HackCU.git`
```
Comment (casassg) [inline README.md]: remember the cd m8
```suggestion
1. `git clone  https://github.com/HackCU/HackCU.git && cd HackCU`
```

Comment (peasant98) [reply README.md]: fixed

File: .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910 (hunk 1 above)
Comment (peasant98) [reply .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910]: fixed

**Review by lucca — APPROVED** [review]
Hand of god guide us

**Pull request merged**